rst_files/mccall_model_with_separation.rst

```diff
@@ -447,17 +434,17 @@ we can create an array for reservation wages for different values of :math:`c`,
 
     for (i, c) in enumerate(c_vals)
         mcm = McCallModel(c = c)
-        w_bar = compute_reservation_wage(mcm)
+        ~, ~, w_bar = solve_mccall_model(mcm)
```

Comment:
?

Reply:
Julian way would be to use keywords to determine optional outputs.

Reply:
We rewrote this function to put everything under one roof (inside a closure), and to return all key objects (`V, U, w_bar`). I think that this is more clear than muddying up the code with `if/else` flags...

Reply:
Alternately, we could return a NT `(V = V, U = U, w_bar = w_bar)`, but I think this is OK for now.

Reply:
That just smells like nargout *cough* MATLAB *cough* If not mistaken there is only one instance besides the default, keyword arguments are not too bad.

Reply:
Having optional outputs is just not important here. I think this is only written this way because of how the discussion around the code was structured. I may change that organization

Reply:
Also, I love named tuple returns when there lots of things returned

Reply:
I made this change to the lecture, and will add this to the style guide.